package.json

```diff
@@ -20,7 +20,11 @@
   },
   "dependencies": {
     "accessible-autocomplete": "^1.6.2",
-    "sass-mq": "^4.0.2"
+    "node-sass": "^4.11.0",
+    "npm": "^6.7.0",
+    "rebuild": "^0.1.2",
```

Comment:
Where does this dependency get used? I can't see it run anywhere in the Docker image or within any `package.json` scripts.